action.yml

```diff
@@ -34,13 +34,21 @@ runs:
       shell: bash
       env:
         TYPE: ${{ steps.gettype.outputs.type }}
+        GITHUB_REPOSITORY: ${{ github.repository }}
       run: |
         major_type=$(echo $TYPE | cut -d "," -f 1)
         minor_type=$(echo $TYPE | cut -d "," -f 2)
+        has_cms6_major="true"
+        if [[ $GITHUB_REPOSITORY == "silverstripe/recipe-plugin" ]] || \
+            [[ $GITHUB_REPOSITORY == "silverstripe/vendor-plugin" ]] || \
+            [[ $GITHUB_REPOSITORY == "silverstripe/silverstripe-graphql-devtools" ]]
+        then
+          has_cms6_major="false"
+        fi
```

Comment:
Have changed to be more dynamic, logic it will now use is to see if there are stable tags for the latest major branch.  If there are none then it will assume that the latest major is a dev branch (just as cms 6 branches are now) and should be skipped

If there are tag for the lastest major branch, then it will assume that the latest major branch is current major branch

This means we should auto switch to running crons on CMS 6 branches when 6.0.0 stable is tagged, though not 6.0.0-beta1